homeassistant/components/tibber/__init__.py

```diff
@@ -20,33 +17,13 @@
     "sensor",
 ]
 
-CONFIG_SCHEMA = vol.Schema(
```

Comment:
We should keep at least:

```py
CONFIG_SCHEMA = cv.deprecated(DOMAIN)
```

It prevents total-breaking configurations of people that have ignored the deprecation.